0_Introduction.ipynb

```diff
@@ -202,7 +203,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Coin Flipping With Turing\n",
+    "### Coin-Flipping With Turing\n",
```

Comment:
```suggestion
    "### Coin-Flipping With Turing\n",
```

This isn't actually an error, I'd prefer the form without a hyphen.

Reply:
Alright, I see the Coin-Flipping above. Never mind, I have unified them without hyphen :)